### ISSUE TITLE
Redundant thumbnail files

### ISSUE THREAD

**Issue by mcrot** [ContactEngineering/topobank#785]
I've seen a case where two thumbnail files are stored for one topography
which doesn't make sense:
```python
In [14]: default_storage.listdir('topographies/user_2/79')
Out[14]: 
([],
 ['UNCD_Coated_Wedge_1_250kx-2_Scale32.699_FIXEDv2p1-squeezed.nc',
  'thumbnail.png',
  'thumbnail_ilnhY9s.png'])
```
We should clean up those files and prevent that this happens in order to save space in the storage backend.

**Comment by mcrot** [comment]
A first analysis shows that all topographies now have thumbnails under "topographies/<id>/thumbnail.png"
in storage and no one is using "user_<user_id>/thumbnail_<id><extra>.png" any more.
So all these thumbnails under "user_<user_id>" can be deleted.

**Comment by mcrot** [comment]
We should carefully think of deleting everything under the "user_<user_id>" paths in storage, since the information
should be saved in the other paths. We need to double check.

**Comment by pastewka** [comment]
I've also seen this and this it is still an issue. Note that `recursive_delete` had a bug that nothing was deleted. This may or may not be fixed with #1012 - will need to observe.

**Comment by pastewka** [comment]
This should be fixed in main, as we are now managing all files through the database.
